creator-node/src/fileManager.js

```diff
@@ -226,7 +223,7 @@ async function saveFileForMultihash (req, multihash, expectedStoragePath, gatewa
     try {
       const ipfs = req.app.get('ipfsLatestAPI')
       const content = fs.createReadStream(expectedStoragePath)
-      for await (const result of ipfs.add(content, { onlyHash: true, timeout: 2000 })) {
+      for await (const result of ipfs.add(content, { onlyHash: true, timeout: 10000 })) {
```

Comment:
this is strictly speaking not related to this change. the reason to include this is we've seen errors where this op would time out and the sync would fail so just wanted to bump this higher